### DISCUSSION TITLE
Drivers for Windows 2000/XP

### DISCUSSION THREAD

**Discussion by SSP6904** [copy/v86#809] [Q&A]
Hi!

I'm wondering how to install the GPU drivers for my Windows 2000 VM on my v86 server..

The BPP is about 8 by default, even though the Windows 2000 on the copy.sh is about 32 with a VBE Driver..

Can anyone tell me how that works or even installing those drivers for Windows 2000/XP?

Thanks alot!

**Comment by dmgnr** [comment]
You can use "Save state" button to get the whole thing i guess

**Reply by SSP6904** [reply]
Well, what I did find was the GPU drivers, which is in the demo on copy.sh, is a standard vbe driver and the color is just 256, which is how you basically get the best graphics possible..

Although it was easy to install on my VM, you just have to look on archive.org for them..

They do work now. Thanks for the tip though! 🙂
